rpcs3/Emu/VFS.cpp

```diff
@@ -196,6 +196,9 @@ bool vfs::unmount(std::string_view vpath)
 
 std::string vfs::get(std::string_view vpath, std::vector<std::string>* out_dir, std::string* out_path)
 {
+	// Initialize vfs_manager if not yet initialized (e.g. g_fxo->reset() was previously invoked)
+	g_fxo->need<vfs_manager>();
```

Comment:
You can also check if it's not initialized instead and call fmt::throw_exception if not.

Reply:
Thanks, done. I also made the check in vfs::retrieve().
Please, feel free to apply any other needed change directly in my PR (you should have permission on that) so the PR can be merged.